projects/cashmere/src/lib/form-field/hc-form-field.component.html

```diff
@@ -1,5 +1,5 @@
-<div [ngClass]="inline ? 'hc-form-field-wrapper-inline' : 'hc-form-field-wrapper'">
-    <span [ngClass]="inline ? 'hc-form-field-label-wrapper-inline' : 'hc-form-field-label-wrapper'">
+<div [ngClass]="{'hc-form-field-wrapper-inline': inline, 'hc-form-field-wrapper': !inline, 'hc-form-field-no-label': !_labelChildren.length}">
+    <span *ngIf="_labelChildren.length" [ngClass]="inline ? 'hc-form-field-label-wrapper-inline' : 'hc-form-field-label-wrapper'">
```

Comment:
Also, `_labelChildren.length` is seen multiple times, maybe just change it to a property and use that?
```ts
get hasLabel() {
  return !!_labelChildren.length;
}
```